src/commands/PublishCommand.js

```diff
@@ -155,6 +155,34 @@ export default class PublishCommand extends Command {
   }
 
   getVersionsForUpdates(callback) {
+    if (this.flags.cdVersion) {
+      // Allows automatic bumping to next semver via cdVersion flag
+      if (this.flags.cdVersion === "patch" ||
+          this.flags.cdVersion === "minor" ||
+          this.flags.cdVersion === "major" ||
+          this.flags.cdVersion === "current"
+      ) {
+        // Warn about possible problems with git and current
+        if (this.flags.cdVersion === "current" && !this.flags.skipGit) {
+          this.logger.info("WARNING! Using --cd-version=current without --skip-git can result in commits without changes and cause errors");
+        }
+        // If the version is independent then send versions
+        if (this.repository.isIndependent()) {
+          const versions = {};
+          this.updates.forEach((update) => {
+            versions[update.package.name] = this.flags.cdVersion === "current" ?
+              update.package.version : semver.inc(update.package.version, this.flags.cdVersion);
```

Comment:
Can you put the ternary operators first, please? It is much easier to read, in my experience.

```
versions[update.package.name] = (this.flags.cdVersion === "current")
  ? update.package.version
  : semver.inc(update.package.version, this.flags.cdVersion);
```